### PR TITLE
test scripts and fix show-features

### LINKED ISSUE #659: `show-features` script broken - test scripts in CI?
```
python scripts/show-features.py Lab11-03.dll -F 0x10001410
Traceback (most recent call last):
  File "scripts/show-features.py", line 216, in <module>
    sys.exit(main())
  File "scripts/show-features.py", line 103, in main
    extractor = capa.main.get_extractor(
TypeError: get_extractor() missing 1 required positional argument: 'should_save_workspace'

```

### PR DESCRIPTION
**Pull request by mr-tz**
closes #659

### Checklist

- [ ] No CHANGELOG update needed

- [ ] No new tests needed

- [x] No documentation update needed

### CODE REVIEW

File: capa/main.py [hunk 1]
```diff
@@ -433,7 +433,7 @@ class UnsupportedRuntimeError(RuntimeError):
 
 
 def get_extractor(
-    path: str, format: str, backend: str, sigpaths: List[str], should_save_workspace, disable_progress=False
+    path: str, format: str, backend: str, sigpaths: List[str], should_save_workspace=False, disable_progress=False
```
Comment (mr-tz) [inline capa/main.py:436]: or should we keep this required?

File: tests/test_scripts.py [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+
+
+@pytest.mark.parametrize(
+    "script,args",
+    [
+        pytest.param(
+            "show-features.py",
+            [get_file_path()],
+        ),
+        pytest.param("bulk-process.py", [get_file_path()]),
```
Comment (mr-tz) [inline tests/test_scripts.py:40]: I tried to parametrize fixtures, but didn't succeed quickly so went with this instead.

**Comment by mr-tz** [comment]
some of these tests take a really long time...

**Review by williballenthin — APPROVED** [review]
nice work

**Comment by williballenthin** [comment]
> some of these tests take a really long time...

i wonder if we pre-generate the .viv with `vivbin -B /path/to/sample` then the tests might go a little faster?

**Comment by williballenthin** [comment]
and yeah these tests are running for more than 2hr of CPU time, which is quite a bit. we might hit quotas at this rate.

**Comment by mr-tz** [comment]
```
CI / Tests in 3.9 on ubuntu-20.04 (pull_request) Cancelled after 360m
CI / Tests in 3.8 on ubuntu-20.04 (pull_request) Cancelled after 360m
```
😮

File: .github/workflows/tests.yml [hunk 3]
```diff
@@ -80,7 +80,10 @@ jobs:
     - name: Install pyyaml
       if: matrix.os == 'ubuntu-20.04'
       run: sudo apt-get install -y libyaml-dev
+    # save workspaces to speed up testing
+    - name: set CAPA_SAVE_WORKSPACE environment variable
+      run: echo "CAPA_SAVE_WORKSPACE=True" >> $GITHUB_ENV
```
Comment (mr-tz) [inline .github/workflows/tests.yml:85]: I hope this speed things up again

**Comment by mr-tz** [comment]
Py 3.8 and 3.9 on Ubuntu run forever... maybe an issue with `bulk-process.py`??

File: .github/workflows/tests.yml (hunk 3 above)
Comment (mr-tz) [reply .github/workflows/tests.yml:85]: confirmed that this works in https://github.com/fireeye/capa/runs/2941515558
`DEBUG:capa:saving workspace`

**Comment by mr-tz** [comment]
Fixed the issue with `bulk-process`

**Review by williballenthin — APPROVED** [review]

**Pull request merged**